WooCommerce/Classes/POS/Presentation/PointOfSaleDashboardView.swift

```diff
@@ -31,15 +31,13 @@ struct PointOfSaleDashboardView: View {
                     .ignoresSafeArea(edges: .bottom)
             }
 
-            if case .card = posModel.paymentState {
-                POSFloatingControlView(showExitPOSModal: $showExitPOSModal,
-                                       showSupport: $showSupport)
-                    .shadow(color: Color.black.opacity(0.12), radius: 4, y: 2)
-                    .offset(x: Constants.floatingControlHorizontalOffset, y: -Constants.floatingControlVerticalOffset)
-                    .trackSize(size: $floatingSize)
-                    .accessibilitySortPriority(1)
-                    .renderedIf(posModel.itemsViewState.containerState != .loading)
-            }
+            POSFloatingControlView(showExitPOSModal: $showExitPOSModal,
```

Comment:
I recall this was added as requested by design, and Android currently does hide the button on cash view entirely. I'll drop a note in Slack as it makes much more sense to always show the floating controls.